packages/codecs/oer/test/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../../../tsconfig.test.json",
+  "compilerOptions": {
+    "composite": true
+  },
+  "references": [
+    {
+      "path": "../src/tsconfig.build.json"
```

Comment:
TypeScript 3.6 should have the API for --build and project references so ts-loader/ts-node/etc can use this ...

Till then, seems have to live with the `pretest` jazz